packages/core/src/components/TextField/Mask.jsx

```diff
@@ -279,7 +282,7 @@ Mask.propTypes = {
  * @param {String} mask
  * @returns {String}
  */
-export function unmask(value, mask) {
+export function unmaskValue(value, mask) {
```

Comment:
I renamed from unmask to unmaskValue to make it consistent with maskValue. It looks like the only reason why it was unmask in Mask.jsx in the first place was because Textfield.jsx imports the function, and then reexports it as unmaskValue. Since the docs already reference the function as unmaskValue, I thought we might as well use the same name everywhere